src/ui/public/management/section.js

```diff
@@ -49,10 +50,18 @@ export class ManagementSection {
     this.url = '';
 
     assign(this, options);
+
+    if (this.parent) {
+      this.capabilityId = camelCase(`${this.parent.id} ${this.id}`);
+    } else {
+      this.capabilityId = null;
+    }
   }
 
   get visibleItems() {
-    return this.items.inOrder.filter(item => item.visible);
+    return this.items.inOrder.filter(item => {
+      return item.visible && uiCapabilities.management[item.capabilityId] !== false;
```

Comment:
Is there a better place to put this?

Reply:
I'm not aware of some other place we should put this, this honestly seems fine.